libcontainer/configs/config.go

```diff
@@ -227,6 +249,23 @@ type Capabilities struct {
 	Ambient []string
 }
 
+func (hooks *Hooks) RunHooks(name HookName, spec *specs.State) error {
+	hooksMap := map[HookName][]Hook{
+		Prestart: hooks.Prestart,
+		CreateRuntime: hooks.CreateRuntime,
+		Poststart: hooks.Poststart,
+		Poststop: hooks.Poststop,
```

Comment:
This file has not been `gofmt`'d, which is why the CI is failing.

Reply:
Sorry about that, fixed :)